### PR TITLE
you should be able to specify who you want to message to on the command line

### PR DESCRIPTION
**Pull request by jeffreybaird**
…nd line

### CODE REVIEW

File: cli/read.rb [hunk 1]
```diff
@@ -1,2 +1,22 @@
 require_relative '../tools/read_hip_chat'
-puts ReadHipChat.new
+
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = "Usage: example.rb [options]"
```
Comment (peterkagey) [inline cli/read.rb:7]: Is this the correct banner?

Comment (jeffreybaird) [inline cli/read.rb:7]: Nope. I got caught C/P

File: cli/send.rb [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+  opts.banner = "Usage: example.rb [options]"
+
+  opts.on('-n', '--username NAME', 'Specify a user name') { |v| options[:user_name] = v }
+
+end.parse!
+
+user_name = options[:user_name]
+
+if user_name
+  WriteHipChat.set_user_name(user_name)
```
Comment (peterkagey) [inline cli/send.rb:15]: Terminating if statements are my favorite thing about Ruby:
`WriteHipChat.set_user_name(user_name) if user_name`

**Comment by peterkagey** [comment]
@jeffreybaird, could you add an example to the README.md?

**Comment by jeffreybaird** [comment]
Yup :+1:

**Comment by peterkagey** [comment]
@jeffreybaird, do it!

**Comment by jeffreybaird** [comment]
On my schedule for today.

**Pull request merged**